test/test-fs.c

```diff
@@ -71,6 +71,7 @@ static int write_cb_count;
 static int unlink_cb_count;
```

Comment:
In this same file, can you update the `fs_null_req` test to also call `uv_fs_mkstemp()` with `NULL` arguments.